src/pages/components/contactUs.js

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment, useState } from 'react';
+import ChatBox from './chatBox';
+
+const ContactUs = () => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
I think we can rename this to `isModalOpen` or `isChatBoxOpen` what do you think?